src/libraries/Microsoft.VisualBasic.Core/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,59 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:Microsoft.VisualBasic.FileIO.TextFieldParser.Dispose</Target>
```

Comment:
Product code issue. Will be fixed with https://github.com/dotnet/runtime/pull/75974.